servers/services/src/main/java/org/projectnessie/services/spi/TreeService.java

```diff
@@ -198,8 +198,6 @@ MergeResponse transplantCommitsIntoBranch(
           String branchName,
       @Valid
           @jakarta.validation.Valid
-          @NotNull
-          @jakarta.validation.constraints.NotNull
```

Comment:
It's better to let the hash validator generate the error messages, they will be more homogenous and more informational.